myst_nbs/case_studies/stochastic_volatility.myst.md

```diff
@@ -6,13 +6,20 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.13.7
 kernelspec:
-  display_name: Python 3 (ipykernel)
+  display_name: pymc_env
   language: python
-  name: python3
+  name: pymc_env
 ---
 
+(notebook_name)=
 # Stochastic Volatility model
 
+:::{post} June 17, 2022
+:tags: time series
+:category: basic
```

Comment:
`basic` is not a valid category: https://docs.pymc.io/en/latest/contributing/jupyter_style.html#first-cell